source/uri-options/uri-options.rst

```diff
@@ -269,6 +277,13 @@ pertaining to URI options apply here.
      - no
      - Amount of time spent attempting to send or receive on a socket before timing out; note that this only applies to application operations, not SDAM
 
+   * - srvServiceName
+     - any string
```

Comment:
Should an empty string be prohibited? Alternatively, should an empty string result in the default value ("mongodb") being used? I assume if we actually allow an empty string as an explicit value we'll get two sequential dots after concatenating, which could be problematic.